# Internal-only patch to remove GCC-specific options from
# configure.ac. These options are currently unsupported by Solaris Studio.
# This patch is not intended for delivery upstream.

--- stunnel-5.18-orig/configure.ac	2015-10-02 12:49:44.966765224 -0700
+++ stunnel-5.18/configure.ac	2015-10-02 12:50:10.306275860 -0700
@@ -71,8 +71,6 @@
 AX_APPEND_COMPILE_FLAGS([-Wconversion])
 AX_APPEND_COMPILE_FLAGS([-Wno-long-long])
 AX_APPEND_COMPILE_FLAGS([-Wno-deprecated-declarations])
-AX_APPEND_COMPILE_FLAGS([-fstack-protector])
-AX_APPEND_COMPILE_FLAGS([-fPIE])
 AX_APPEND_COMPILE_FLAGS([-D_FORTIFY_SOURCE=2])
 AX_APPEND_LINK_FLAGS([-fPIE -pie])
 AX_APPEND_LINK_FLAGS([-Wl,-z,relro])
